e2e/internal/client/outputs.go

```diff
@@ -86,8 +87,8 @@ type SvcShowRoutes struct {
 	URL         string `json:"url"`
 }
 
-// SvcShowServiceDiscoveries contains serialized service discovery info for an service.
-type SvcShowServiceDiscoveries struct {
+// SvcShowServiceEndpoints contains serialized endpoint info for an service.
```

Comment:
tiny nit (not your typo):
```suggestion
// SvcShowServiceEndpoints contains serialized endpoint info for a service.
```